api.go

```diff
@@ -0,0 +1,45 @@
+package slack
```

Comment:
i dislike this file name. too generic. move it into the file with the client definition if you're looking for somewhere to push it.